embc-app/Services/Referrals/IReferralsService.cs

```diff
@@ -5,7 +5,9 @@ namespace Gov.Jag.Embc.Public.Services.Referrals
 {
     public interface IReferralsService
     {
-        Task<string> GetReferralHtmlPages(ReferralsToPrint printReferrals);
+        Task<byte[]> GetReferralPdfsAsync(ReferralsToPrint printReferrals);
+
+        Task<string> GetReferralHtmlPagesAsync(ReferralsToPrint printReferrals);
```

Comment:
is this part of the interface? Is it used?

Reply:
Should have removed that as well.